app/templates/components/new-myreport.hbs

```diff
@@ -28,7 +28,7 @@
     <p>
       {{t 'dashboard.associatedWith'}}
       <select onchange={{action "changePrepositionalObject" value="target.value"}}>
-        <option value=null selected={{is-equal null currentPrepositionalObject}}>
+        <option value='' selected={{is-equal null currentPrepositionalObject}}>
```

Comment:
`value=null` came through as string value "null". which to me feels even _weirderer_ than evaluating the truthiness of a blank string.